Esp32PmSensor/Esp32PmSensor.ino

```diff
@@ -19,13 +19,12 @@
 
 #include "sds011.h"
 
-// This EUI must be in BIG-ENDIAN format, so least-significant-byte first.
+// This EUI must be in BIG-ENDIAN format, most-significant byte (MSB).
+// In TTN console switch format with the <> button and choose msb with ⇄
```

Comment:
Thanks, that was a remnant of an earlier incomplete change.

You are right about the text being incorrect, but I don't like the new line either.
Please remove the new comment about switching the format, as that is likely to lead to people getting confused about it.

The intention was that keys should be copied from the TTN console with the least amount of "fiddling" with format buttons, etc.

Reply:
No, problem. Simplified in 4dcd392f99ab79e1100472d19f0a5f926af07f39